cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -130,6 +130,12 @@ API Changes
       }
     }
   
+- :cask-issue:`CDAP-7544` - The :ref:`Metadata HTTP RESTful API
+  <http-restful-api-metadata-searching>` has been modified to support sorting and
+  pagination. To do so, the API now uses cursors and chunking, and the format of the results
```

Comment:
To do so, the API uses additional parameters ``sort``, ``offset``, ``limit``, ``numCursors`` and ``cursor``....